tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/ILanguageSpecificResolver.cs

```diff
@@ -13,4 +14,12 @@ public interface ILanguageSpecificResolver<T> where T : class
     /// <param name="ct">Cancellation token.</param>
     /// <returns>The resolved service instance, or null if no matching service is found.</returns>
     public Task<T?> Resolve(string packagePath, CancellationToken ct = default);
+
+    /// <summary>
+    /// Resolves the language-specific service implementation for a given Sdk language.
+    /// </summary>
+    /// <param name="language">A language identifier.</param>
+    /// <param name="ct">Cancellation token.</param>
+    /// <returns>A list of resolved service instances (null for languages that could not be resolved).</returns>
+    public Task<T?> Resolve(SdkLanguage language, CancellationToken ct = default);
```

Comment:
Based on the documentation comment (docstring) the return type should be a list of service instances. To match the docstring, the return type should be: `Task<List<T?>>`